### PR TITLE
Server migrate es6

### PR DESCRIPTION
**Pull request by asclines**
Upgrades the server code to ES6. That is all.

TODO:
- Update tests to use new testing style

### CODE REVIEW

File: test/server/dummy-data.js [hunk 1]
```diff
@@ -1,15 +1,16 @@
-module.exports = {
-  auth: {
+export let auth = {
     "patty.lastname@ttu.edu": "patty123",
     "john.doesnt@ttu.edu": "john234",
     "sally.does@ttu.edu": "324sally"
-  },
-  emailToUid: {
+  }
```
Comment (jmtaber129) [inline test/server/dummy-data.js]: Please fix indentation.  Also, this statement should end with a semicolon after the closing brace.

File: test/server/dummy-data.js [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
+
+export let emailToUid = {
     "patty.lastname@ttu.edu": "QveDEhTHWSgVbnL4NXrdD6rSvns1",
     "john.doesnt@ttu.edu": "hMXRXnHKQdbGmX9bwaZntRaJER03",
     "sally.does@ttu.edu": "k8fcP5IgEVcuVVaOTQWduYwheur1"
-  },
-  users: {
+  }
```
Comment (jmtaber129) [inline test/server/dummy-data.js]: Indentation and semicolon.

File: test/server/dummy-data.js [hunk 3]
```diff
@@ -34,20 +35,21 @@ module.exports = {
       role: "STUDENT",
       studentId: "110451"
     }
-  },
-  reports: {
+  }
```
Comment (jmtaber129) [inline test/server/dummy-data.js]: Indentation and semicolon.

File: test/server/dummy-data.js [hunk 4]
```diff
@@ -34,20 +35,21 @@ module.exports = {
...
+
+export let reports = {
     "QdbGmX9bwaOTQWduYw": {
       type: "event",
       student: "110451",
       event: "someevent"
     }
-  },
-  events: {
+  }
```
Comment (jmtaber129) [inline test/server/dummy-data.js]: Indentation and semicolon.

File: test/server/dummy-data.js [hunk 5]
```diff
@@ -34,20 +35,21 @@ module.exports = {
...
+
+export let events = {
     "QWduYwX": {
       "datetime": "2017:05:08:07:00",
       "location": "Silicon Valley",
       "title": "GTC",
       "description": "GPU Technology Conference. The Largest and most important event of the year for GPU developers."
     }
   }
```
Comment (jmtaber129) [inline test/server/dummy-data.js]: Indentation and semicolon.

**Comment by asclines** [comment]
@jmtaber129 requesting new review now that request changes are fixed

**Pull request merged**